libs/datamodel/core/tests/reformat/reformat.rs

```diff
@@ -74,6 +74,146 @@ model Post {
     assert_reformat(input, expected);
 }
 
+#[test]
+fn format_should_enforce_order_of_field_directives() {
```

Comment:
It might a good idea to add a field that has (almost) all directives to ensure the ordering is correct.